fakeserver/testdata/schemas/test.yml

```diff
@@ -0,0 +1,7 @@
+serializer_version: 1
```

Comment:
My thought is, if we're using `ioutil.TempDir`, we can cp this in there, or generate it entirely. Thoughts?